From 3785b25fd05fffcbfe0f28affe87eeb4518dcaca Mon Sep 17 00:00:00 2001
Date: Thu, 27 Jun 2024 18:59:56 +0300
Subject: [PATCH] Fix jquery loading in the hidden frame

---
 vendor/jquery/jquery.js            | 4 ++++

diff --git a/vendor/jquery/jquery.js b/vendor/jquery/jquery.js
index 1a86433c22..b064264556 100644
--- a/vendor/jquery/jquery.js
+++ b/vendor/jquery/jquery.js
@@ -6409,6 +6409,10 @@ var rboxStyle = new RegExp( cssExpand.join( "|" ), "i" );
 				documentElement.removeChild( table );
 			}
 			return reliableTrDimensionsVal;
+		},
+		forceStyleTests: function() {
+			div = document.createElement( "div" );
+			computeStyleTests();
 		}
 	} );
 } )();
--
2.46.0.windows.1

